plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -170,6 +170,31 @@ public void testCreateAndDropTableWithSpecialCharacterName(String tableName)
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    // TODO remove this test after refactoring testCreateAndDropTableWithSpecialCharacterName
+    @Override
+    @Test(dataProvider = "testTableNameDataProvider")
+    public void testRenameColumnName(String columnName)
+    {
+        super.testRenameColumnName(columnName);
+    }
+
+    // TODO move this test to BaseConnectorTest
+    @Test(dataProvider = "testTableNameDataProvider")
+    public void testRenameFromToTableWithSpecialCharacterName(String tableName)
+    {
+        // Until https://github.com/trinodb/trino/issues/17 the table name is effectively lowercase
+        tableName = tableName.toLowerCase(ENGLISH);
+        String tableNameInSql = "\"" + tableName.replace("\"", "\"\"") + "\"";
```

Comment:
This way you test only lowercase table names and test won't be updated when #17 is done

Note that in https://github.com/trinodb/trino/blob/19394a63cd41f7a80b1d023d056b7238f2bc1869/plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java#L162-L164 we have different order of statements, intentionally.